meteor/server/api/blueprints/context/lib.ts

```diff
@@ -26,12 +26,12 @@ export const IBlueprintPieceSampleKeys = allKeysOfObject<IBlueprintPiece>({
 	content: true,
 	transitions: true,
 	lifespan: true,
-	adlibPreroll: true,
+	prerollDuration: true,
 	toBeQueued: true,
 	expectedPlayoutItems: true,
-	adlibAutoNext: true,
-	adlibAutoNextOverlap: true,
-	adlibDisableOutTransition: true,
+	// adlibAutoNext: true,
+	// adlibAutoNextOverlap: true,
```

Comment:
remove these?

Reply:
theyve already been removed in r39, will be tidied as part of merging r39 back into this branch